src/ssh_via_subprocess.py

```diff
@@ -0,0 +1,81 @@
+import subprocess
+import json
+import asyncio
+from shlex import join, quote
+
+
+async def run_remote_commands_for_data(username, password, host, url_to_curl, port='22'):
+    command = "sshpass -p {} ssh -p {} -o StrictHostKeyChecking=no {}@{} 'curl -s {}'".format(quote(password), quote(str(port)), quote(username), quote(host), quote(url_to_curl))
```

Comment:
@abhishekkushwaha4u you are still using the same formatting way to create the subprocess command?

Reply:
I did it with official docs, don't worry, this is safe now.